tests/fixtures/policies/dummy_policy.py

```diff
@@ -32,7 +33,7 @@ def get_action(self, observation):
             dict: Distribution parameters.
 
         """
-        return self.action_space.sample(), dict(dummy='dummy')
+        return self.action_space.sample(), dict(dummy='dummy', mean=0.)
```

Comment:
don't you need to call `super().__init__(...)` for `self.action_space` exist in this class?

Reply:
I guess the class just inherits the `action_space` property, which uses `self.env_spec`, which is defined for the `DummyPolicy` so everything works. The `__init__` for the base Policy is blank, so it doesn't do anything interesting. 
The CI passes, but if you want, I can add the `super()` call.